account/views.py

```diff
@@ -351,6 +357,22 @@ def closed(self):
         }
         return self.response_class(**response_kwargs)
 
+    def account_approval_required_response(self):
+        if self.request.is_ajax():
+            template_name = self.template_name_admin_approval_ajax
```

Comment:
This is misspelled - it should be `template_name_admin_approval_sent_ajax`.

Reply:
thanks for the review! should be fixed now.

Reply:
Cool, thanks! Hopefully I can be a maintainer for this repository soon.

One thing I'd eventually like to do is write some tests for this project. If you have the time, can you write some tests for this feature?

Reply:
Tests already included: https://github.com/pinax/django-user-accounts/pull/256/files#diff-15a330fd5d8811681fa96af031169262fef751ecbe6bf7b8cfa39438e6a1d47d